crates/sui-types/src/unit_tests/utils.rs

```diff
@@ -36,7 +36,7 @@ pub struct TestData {
     pub zklogin_inputs: String,
     pub kp: String,
     pub pk_bigint: String,
-    pub randomness: String,
+    pub salt: String,
```

Comment:
this meaningfully changes the address seed value hence changes public identifier. so usable by multisig test that requires no dup in public identifiers.